### ISSUE TITLE
Logging configuration is overwritten if pytezos module is included prior to main configuring the logger.

### ISSUE THREAD

**Issue by steampoweredtaco** [baking-bad/pytezos#287]
This code
```python
import logging
from pytezos import pytezos

logging.basicConfig(level=logging.INFO, format='%(asctime)s - %(name)s - %(levelname)s - {%(pathname)s:%(lineno)d} - %(message)s')
log = logging.getLogger('main')
log.info("funky logging")
```

Produces this:
`funky logging`

However, importing pytezos should not have side effects on the basic configuration of the logger as standard for modules and it should produce something like this:
`2022-03-05 03:57:00,302 - main - INFO - {<ipython-input-4-2ba778774d25>:1} - funky logging`

The workaround, which makes the imports messy, is to only import pytezos after the basic config of the logger is configured like this:
```python
import logging

logging.basicConfig(level=logging.INFO, format='%(asctime)s - %(name)s - %(levelname)s - {%(pathname)s:%(lineno)d} - %(message)s')
log = logging.getLogger('main')
from pytezos import pytezos

log.info("funky logging")
```

I am guessing this is likely to prevent the correct logger level to be set as well if the basic config method is used in someone's main program without the workaround.

**In summary**:  if `from pytezos import pytezos` comes prior to `logging.basicConfig` in a main program pytezos prevents the new configuration.

**Thoughts/Fix**
I don't think this is a regression of #209 

 I'm guessing that the intended behavior is if pytezos module is used as a cli that the logging configs should be overridden and the current behavior is unintended?

If so, then https://github.com/baking-bad/pytezos/blob/0407e370ee7551fe7c819def03c6adedd2abcb5f/src/pytezos/logging.py#L27-L28 should not be under an if case here but simply used before creating the cli here https://github.com/baking-bad/pytezos/blob/0407e370ee7551fe7c819def03c6adedd2abcb5f/src/pytezos/__main__.py#L4
